test/unit/test_descr_skulpt.py

```diff
@@ -19,7 +19,7 @@ def test(self):
         self.assertTrue(str(test).startswith('<function test'))
 
         bound_no_type = test.__get__(4)
-        self.assertEqual(str(bound_no_type), '<bound method ?.test of 4>')
```

Comment:
these are the python2 tests we should probably make these changes behind a flag

Reply:
Good point!

i've reverted these checks so that bound methods no type are `'?'` as before. (This will be the case in either `py3` or `py2`)

Which I figured is preferable to a `future` flag at the moment. Mostly because skulpt implementation of `methods` is entirely based on `py2` and to move to py3 will require more than a plaster over `$r` which is what I was initially trying to do with this.

Reply:
resolved